assets/src/application/ui/display/entityDetailsPanel/Sold.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { addQueryArgs } from '@wordpress/url';
+import { __ } from '@wordpress/i18n';
+
+import { ADMIN_ROUTES } from '@sharedConstants/adminRoutes';
+import getAdminUrl from '@sharedServices/utils/url/getAdminUrl';
+import { EntityDbId } from '@dataServices/types';
+import useConfig from '@appServices/config/useConfig';
+import useEventId from '@edtrServices/apollo/queries/events/useEventId';
+
+import { Link } from '@application/ui/input';
+
+import './style.scss';
+
+interface Props {
+	dbId: EntityDbId;
+	sold?: number;
+	type: 'date' | 'ticket';
+}
+
+const Sold: React.FC<Props> = ({ sold = 0, type, ...props }) => {
```

Comment:
before going with simple `Sold`, this was named `EntitySoldDetail`, but that prefix and suffix was just plain noise and wasn't aiding in any way

Reply:
ya I think that's a nice change 👍